doc/oemof_solph.rst

```diff
@@ -432,11 +432,53 @@ Furthermore, an efficiency for loading, unloading and a capacity loss per time i
         capacity_loss=0.001, nominal_capacity=50,
         inflow_conversion_factor=0.98, outflow_conversion_factor=0.8)
 
+
+Using the investment object with the GenericStorage component
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The `GenericInvestmentStorageBlock` enables the investment mode for storages to optimise dispatch and investment. It is 
+based on the `GenericStorage` component and there are two main investment possibilities.
+
+    *	Invest into the flow parameters e.g. a turbine or a pump
+    *	Invest into capacity of the storage  e.g. a basin or a battery cell
+    
+As an addition to other flow-investments, the storage class implements the possibility to couple or decouple the flows 
+with the capacity of the storage. 
+Three parameters are responsible for connecting the flows and the capacity of the storage:
+
+    *	' `invest_relation_input_capacity` ' fixes the inflow to the capacity. A ratio of ‘1’ means that the storage can be emptied within one time-period. 
```

Comment:
> fixes the inflow to the capacity

In my opinion the relation between the investment variable of the inflow and the investment variable of the capacity is set.

Reply:
That is correct I suppose, but isn't that the same in the end? I think the best compromise is to say "the Investment of... is fixed".

Reply:
I do not know what is obvious for new users, but it would help to define some words in the beginning and use them later on. For some people Investment is an amount of money and not the installed capacity.

We could write something like  "The Investment is the value of the variable for the 'nominal_value' (installed_capacity) in the investment mode.".